framework_tvb/tvb/core/entities/storage/datatype_dao.py

```diff
@@ -245,7 +245,7 @@ def count_datatypes_generated_from(self, datatype_gid):
         count = 0
         try:
             count = self.session.query(DataType).join(Operation
-                                        ).filter(Operation.parameters.ilike('%' + datatype_gid + '%')).count()
+                                        ).filter(Operation.view_model_gid.ilike('%' + datatype_gid + '%')).count()
```

Comment:
After this change, this counter becomes useless. We were expecting to match a DataType child, now we compare with a ViewModel.gid (it will never return True). We should review all its usages, and try to replace the logic

Reply:
After analyzing where these are used, as well as discussing it with @popaula937 the proposal here is the following:
- datatypes overlay: simply remove the TAB "Derived DataTypes". Make sure entire removal: from the HTML Jinja, linked JS code, if any, code in python (down to these DAO calls). Also double check if these are not used elsewhere.
- document in Jira in an explicit comment that we do this removal, as well as describe a potential solution (with a `DataType.from_other_datatypes_gids = Column(String) # JSON with datatypes inputs in the operation.view_model used when producing the current DT`